ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -34,13 +35,13 @@ import (
 // Refactor to import these constants instead of re-defining them here.
 const (
 	// ecsLocalLabelKey is the Docker object label associated with containers created with "ecs-cli local".
-	ecsLocalLabelKey = "ECSLocalTask"
+	ecsLocalLabelKey = "ecsLocalTask"
 
-	// taskDefinitionARNLabelKey is the Docker object label present if the container was created with a task def ARN.
-	taskDefinitionARNLabelKey = "taskDefinitionARN"
+	// taskDefinitionLabelKey is the Docker object label present if the container was created with an ARN or task family.
+	taskDefinitionLabelKey = "ecsTaskDefinition"
 
 	// taskFilePathLabelKey is the Docker object label present if the container was created from a file.
-	taskFilePathLabelKey = "taskFilePath"
+	taskFilePathLabelKey = "ecsTaskFilePath"
```

Comment:
idea: Do we really need separate labels for TD Arn vs File? Instead you could just have "ecsTaskDefinition" and the value could be either the Arn or File. You can always differentiate those because an Arn has to start with "arn", and a full file path can't (on Windows or Unix).

Reply:
That's a great idea, I'll change it to that! Since the two (arn/family and file) are mutually exclusive, they can definitely appear under the same column.

Reply:
Done!